examples/pcc-dialin-server/simple-python/server.py

```diff
@@ -0,0 +1,197 @@
+"""
+
+
+@app.get("/")
+async def read_root():
+    return {"message": "Hello, World!"}
+
+
+@app.post("/api/dial")
+async def dial(request: RoomRequest, raw_request: Request):
+    logger.info("Incoming request to /dial:")
+    logger.info(f"Headers: {dict(raw_request.headers)}")
+    raw_body = await raw_request.body()
+    raw_body_str = raw_body.decode()
+    logger.info(f"Raw body: {raw_body_str}")
+    logger.info(f"Parsed body: {request.dict()}")
+
+    # calculate signature and compare/verify
+    hmac_secret = os.getenv("PINLESS_HMAC_SECRET")
+    timestamp = raw_request.headers.get("x-pinless-timestamp")
+
+    if timestamp:
+        signature = timestamp + "." + raw_body_str
+        logger.debug(f"Signature: {signature}")
+
+        base64_decoded_secret = base64.b64decode(hmac_secret)
+        computed_signature = base64.b64encode(
+            hmac.new(base64_decoded_secret, signature.encode(), "sha256").digest()
+        ).decode()
+
+        if computed_signature != raw_request.headers.get("x-pinless-signature"):
+            logger.error("Invalid signature")
+            raise HTTPException(status_code=401, detail="Invalid signature")
+
+    if request.test == "test":
+        logger.debug("Test request received")
+        return {"status": "success", "message": "Test request received"}
+
+    dialin_settings = None
+    # these fields are camelCase in the request
+    required_fields = ["To", "From", "callId", "callDomain"]
+    if all(
+        field in request.dict() and request.dict()[field] is not None for field in required_fields
+    ):
+        # transform from camelCase to snake_case because daily-python expects snake_case
+        dialin_settings = {
+            "From": request.From,
+            "To": request.To,
+            "call_id": request.callId,
+            "call_domain": request.callDomain,
+            # transform from camelCase to snake_case
+        }
+        logger.debug(f"Populated dialin_settings from request: {dialin_settings}")
+
+    daily_room_properties = {
+        "enable_dialout": request.dialout_settings is not None,
+    }
+
+    if dialin_settings is not None:
+        sip_config = {
+            "display_name": "dialin",
+            "sip_mode": "dial-in",
+            "num_endpoints": 2 if request.call_transfer is not None else 1,
+        }
+        daily_room_properties["sip"] = sip_config
+
+    # Setting default expiry to 5 minutes from now
+    daily_room_properties["exp"] = int(time.time()) + (5 * 60)
+
+    logger.debug(f"Daily room properties: {daily_room_properties}")
+    payload = {
+        "createDailyRoom": True,
+        "dailyRoomProperties": daily_room_properties,
+        "body": {
+            "dialin_settings": dialin_settings,
+            "dialout_settings": request.dialout_settings,
+            "voicemail_detection": request.voicemail_detection,
+            "call_transfer": request.call_transfer,
+        },
+    }
+
+    daily_api_key = os.getenv("DAILY_API_KEY")
```

Comment:
I think these should be the Pipecat Cloud API key

```suggestion
    pcc_api_key = os.getenv("PIPECAT_CLOUD_API_KEY")
```